app/components/custom_list/index.js

```diff
@@ -35,10 +36,11 @@ export default class CustomList extends PureComponent {
     };
 
     static defaultProps = {
+        canRefresh: true,
+        isLandscape: false,
```

Comment:
Is this set to support Unit Testing, since the prop isRequired?

Reply:
I wouldn’t worry, that in fact I plan to remove in the near future